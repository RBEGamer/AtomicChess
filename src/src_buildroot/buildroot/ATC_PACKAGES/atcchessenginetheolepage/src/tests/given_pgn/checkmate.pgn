[Event "?"]
[Site "Moscow RUS"]
[Date "1918.??.??"]
[Round "?"]
[White "Alekhine, Alexander A"]
[Black "Rabinovich Abram"]
[Result "1-0"]
[ECO "C42m"]

1. e2-e4 e7-e5 2. Ng1-f3 Ng8-f6 3. Nf3xe5 d7-d6 4. Ne5-f3 Nf6xe4 5. Nb1-c3
d6-d5 6. Qd1-e2 Bf8-e7 7. Nc3xe4 d5xe4 8. Qe2xe4 O-O 9. Bf1-c4 Be7-d6 10.
O-O Rf8-e8 11. Qe4-d3 Nb8-c6 12. b2-b3 Qd8-f6 13. Bc1-b2 Qf6xb2 14.
Nf3-g5 Bc8-e6 15. Bc4xe6 f7xe6 16. Qd3xh7+ Kg8-f8 17. Ra1-e1 Qb2-f6 18.
Qh7-h5 Kf8-g8 19. Re1-e3 Bd6-f4 20. Qh5-h7+ Kg8-f8 21. Qh7-h8+ Kf8-e7 22.
Re3xe6+ Qf6xe6 23. Qh8xg7+ Ke7-d6 24. Ng5xe6 Re8xe6 25. d2-d4 Ra8-e8 26.
c2-c4 Re8-e7 27. Qg7-f8 Re6-e4 28. Qf8-f5 Re4xd4 29. c4-c5# 1-0

