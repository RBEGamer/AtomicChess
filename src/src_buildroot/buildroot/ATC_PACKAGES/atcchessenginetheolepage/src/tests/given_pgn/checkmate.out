[move]	board::PieceType: 5
[move]	board::PieceType: 5
[move]	board::PieceType: 4
[move]	board::PieceType: 4
[move]	board::PieceType: 4
[piece_taken]	 piece: 5 file: 5 rank: 5
[move]	board::PieceType: 5
[move]	board::PieceType: 4
[move]	board::PieceType: 4
[piece_taken]	 piece: 5 file: 5 rank: 4
[move]	board::PieceType: 4
[move]	board::PieceType: 5
[move]	board::PieceType: 1
[move]	board::PieceType: 3
[move]	board::PieceType: 4
[piece_taken]	 piece: 4 file: 5 rank: 4
[move]	board::PieceType: 5
[piece_taken]	 piece: 4 file: 5 rank: 4
[move]	board::PieceType: 1
[piece_taken]	 piece: 5 file: 5 rank: 4
[move]	board::PieceType: 0
[kingside castling]	color: 1
[move]	board::PieceType: 3
[move]	board::PieceType: 3
[move]	board::PieceType: 0
[kingside castling]	color: 0
[move]	board::PieceType: 2
[move]	board::PieceType: 1
[move]	board::PieceType: 4
[move]	board::PieceType: 5
[move]	board::PieceType: 1
[move]	board::PieceType: 3
[move]	board::PieceType: 1
[piece_taken]	 piece: 3 file: 2 rank: 2
[move]	board::PieceType: 4
[move]	board::PieceType: 3
[move]	board::PieceType: 3
[piece_taken]	 piece: 3 file: 5 rank: 6
[move]	board::PieceType: 5
[piece_taken]	 piece: 3 file: 5 rank: 6
[move]	board::PieceType: 1
[piece_taken]	 piece: 5 file: 8 rank: 7
[check]	: color: 1
[move]	board::PieceType: 0
[move]	board::PieceType: 2
[move]	board::PieceType: 1
[move]	board::PieceType: 1
[move]	board::PieceType: 0
[move]	board::PieceType: 2
[move]	board::PieceType: 3
[move]	board::PieceType: 1
[check]	: color: 1
[move]	board::PieceType: 0
[move]	board::PieceType: 1
[check]	: color: 1
[move]	board::PieceType: 0
[move]	board::PieceType: 2
[piece_taken]	 piece: 5 file: 5 rank: 6
[check]	: color: 1
[move]	board::PieceType: 1
[piece_taken]	 piece: 2 file: 5 rank: 6
[move]	board::PieceType: 1
[piece_taken]	 piece: 5 file: 7 rank: 7
[check]	: color: 1
[move]	board::PieceType: 0
[move]	board::PieceType: 4
[piece_taken]	 piece: 1 file: 5 rank: 6
[move]	board::PieceType: 2
[piece_taken]	 piece: 4 file: 5 rank: 6
[move]	board::PieceType: 5
[move]	board::PieceType: 2
[move]	board::PieceType: 5
[move]	board::PieceType: 2
[move]	board::PieceType: 1
[move]	board::PieceType: 2
[move]	board::PieceType: 1
[move]	board::PieceType: 2
[piece_taken]	 piece: 5 file: 4 rank: 4
[move]	board::PieceType: 5
[mat]	: color: 1
[game_finished]	
